portfolio/src/main/webapp/style.css

```diff
@@ -131,33 +131,37 @@ ul {
   text-decoration: none;
 }
 
-.fa-github, 
+.fa-github,
 .fa-github:visited {
   color: black;
 }
 
-.fa-linkedin, 
+.fa-linkedin,
 .fa-linkedin:visited {
   color: #0072b1;
 }
 
+.profile-text {
+  display: none;
+}
+
 /* Comments Section */
 
 #login-container {
-  background: rgba(255,255,255,.4);
+  background: rgba(255, 255, 255, 0.4);
   padding: 5px 5px 10px 5px;
   text-align: center;
 }
 
 #login-logout-link {
   color: inherit;
-  padding: .5px 7px 1px;
+  padding: 0.5px 7px 1px;
   text-decoration: none;
 }
 
 #update-name,
 #post-comment {
-  background: rgba(255,255,255,.4);
+  background: rgba(255, 255, 255, 0.4);
```

Comment:
Does your linter complain about this?
According to https://google.github.io/styleguide/htmlcssguide.html#Leading_0s leading 0s should be omitted.

Reply:
I didn't see any message from the linter about it. Originally, there weren't leading zeros but running prettier added them.